packages/rxjs-firebase/src/operators/catchDocumentSnapshotError.ts

```
@@ -0,0 +1,32 @@
+import { type DocumentData } from '@firebase/firestore'
```

ESLint

[error] Filename is not in kebab case. Rename it to `catch-document-snapshot-error.ts`.

(unicorn/filename-case)